### PR TITLE
Added HTML AST parser support

### PR DESCRIPTION
**Pull request by marcelaraujo**

### CODE REVIEW

**Comment by marcelaraujo** [comment]
@stephanebachelier Done!

**Comment by stephanebachelier** [comment]
@marcelaraujo don't know why it fails on appveyor.

I've seen that some changes have been made on master. I will see if it can help fix this issue.

File: test/test-file.js [hunk 1]
```diff
@@ -16,15 +16,16 @@ describe('File', function () {
     assert.ok(file.blocks.length, 2);
   });
 
-  it('should *not* skip blank lines', function () {
+  it('should skip blank lines', function () {
     var filename = __dirname + '/fixtures/block_with_empty_line.html';
```
Comment (stephanebachelier) [inline test/test-file.js]: @marcelaraujo Could you rebase on dev head and change `__dirname + '/fixtures/block_with_empty_line.html'` to 
`path.join(__dirname, '/fixtures/block_with_empty_line.html')`.
This should help us keep in sync with master.

**Comment by marcelaraujo** [comment]
I found.

;)

**Pull request closed without merging**